### ISSUE TITLE
shap summary plot has different class colors across train and test data 

### ISSUE THREAD

**Issue by lachlanmichalski** [shap/shap#1444]
Hi I use the following to create a bar plot of training and test shapley values:

plt.figure(figsize=(height, width))
plt.subplot(1,2,1)
shap.summary_plot(shap_values_train, X_display_train, 
                  max_display=len(X_display_train.columns),
                  plot_type="bar",
                  plot_size=(width, height), show=False)
plt.title('{} {} training set'.format(sample, model_name), fontsize=14)
plt.xlabel('mean(|SHAP values|) (average impact on model output magnitude)',fontsize=12)
   
plt.subplot(1,2,2)
shap.summary_plot(shap_values_test, X_display_test, 
              max_display=len(X_display_test.columns),
              plot_type="bar",
              plot_size=(width, height), show=False) 
plt.title('{} {} test set'.format(sample, model_name), fontsize=14)
plt.xlabel('mean(|SHAP values|) (average impact on model output magnitude)',fontsize=12)

plt.subplots_adjust(wspace=0.45)
plt.show()

What I find is that the color of the classes is different across the training and testing outputs when the inputted data looks the same. Is there a way to fix this? It is just slightly annoying for interpretation with the same class having different colors (e.g. class 0 and class 1 change color)

![image](https://user-images.githubusercontent.com/70327616/93654556-16628000-fa16-11ea-940e-7d83b3ad7f9a.png)

**Comment by Saharati90** [comment]
Have you tried class_names=model.classes_  ?
